### DISCUSSION TITLE
How should we converge work toward the right set of branches?

### DISCUSSION THREAD

**Discussion by bnord** [deepskies/DeepSZSim#55] [Ideas]
For background, here are few standards for most code repositories
1. `main`: only used as the branch through which to release the code. That is, never commit/push code to main.
2. `development`: most of the time, branches should come off this branch. We're in a slightly more complex situation, and we should try to clean up our branches so that we're usually branching off development. 
3. `dev-pipeline`: I'm reorganizing the repository and filling in the missing elements of the workflow with this branch.

We have multiple branches that have come from other branches, which may soon be stale.

For now, everyone should only commit/push to the branches they're working on. Please do not push to `main`, `development`, or `dev-pipeline`.

**Comment by evavagiakis** [comment]
Right now, the most up-to-date code I have been working with is in https://github.com/deepskies/SimSZ/tree/dev-massdist. Any code from other branches in SimSZ is likely stale. The students are working with the code from dev-massdist at the moment.

**Reply by bnord** [reply]
awesome. 

I'm thinking folks should push back to dev-massdist (via a PR), and then we can push dev-massdist to dev-pipeline or development, depending on what that's ready. 

What do you think?

**Reply by evavagiakis** [reply]
Sounds good to me
